examples/game/breakout.rs

```diff
@@ -1,16 +1,54 @@
+//! A simplified implementation of the classic game "Breakout"
+
 use bevy::{
     core::FixedTimestep,
+    math::{const_vec2, const_vec3},
     prelude::*,
     sprite::collide_aabb::{collide, Collision},
 };
 
-/// An implementation of the classic game "Breakout"
+// Sets the amount of time that should elapse between each physics step
 const TIME_STEP: f32 = 1.0 / 60.0;
+
+// These constants are in `Transform` units:
+// with the default 2D camera they correspond 1:1 with screen pixels
+// These macros are needed as functions that operate on floats cannot be constant in Rust :(
+const PADDLE_HEIGHT: f32 = -215.0;
+const PADDLE_SIZE: Vec3 = const_vec3!([120.0, 30.0, 0.0]);
+const PADDLE_SPEED: f32 = 500.0;
+const PADDLE_BOUNDS: f32 = 380.0;
+
+// We set the z-value of the ball to 1 so it renders on top in the case of overlapping sprites
+const BALL_STARTING_POSITION: Vec3 = const_vec3!([0.0, -50.0, 1.0]);
+const BALL_SIZE: Vec3 = const_vec3!([30.0, 30.0, 0.0]);
+const BALL_SPEED: f32 = 400.0;
+// This vector is later extended into three dimensions and then normalized
```

Comment:
this comment feels extraneous. Especially with how disconnected it is from where it actually is done.

Reply:
```suggestion
```